utilities/transactions/write_prepared_transaction_test.cc

```diff
@@ -3497,6 +3497,31 @@ TEST_P(WritePreparedTransactionTest, WC_WP_WALForwardIncompatibility) {
   CrossCompatibilityTest(WRITE_PREPARED, WRITE_COMMITTED, !empty_wal);
 }
 
+TEST_P(WritePreparedTransactionTest, IngestFile) {
```

Comment:
What are the minimum conditions to trigger the test? two_write_queue and IngestExternalFile? If yes, can we move the test to ./db/external_sst_file_test.cc?

Reply:
ok